Base/usr/share/man/man1/su.md

```diff
@@ -0,0 +1,41 @@
+## Name
+
+su - run a command with substitute user
+
+## Synopsis
+
+```
+$ su [user]
+```
+
+## Description
+
+**su** allows commands to be run with a substitute user.
+
+When called with no user-specified, **su** defaults to running an interactive shell as *root*. 
+
+## Options
+
+- **--help**
+
+  Display help message and exit.
+
+- **--version**
```

Comment:
Arguments and options are enclosed in code formatting (backticks \`\` ) and the description can be on the same line. Look at other manpages in the same category, like bt(1) or asctl(1), to keep it consistent.